x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/alerts/trial_license_complete_tier/document_level_security.ts

```diff
@@ -74,7 +74,7 @@ export default ({ getService }: FtrProviderContext) => {
   const esArchiver = getService('esArchiver');
   const security = getService('security');
 
-  describe('find alert with/without doc level security', () => {
```

Comment:
This test has likely not been picked up as it did not have a tag. I think this test should apply to serverless as well.

@paulewing can you confirm my thinking that this is just a Platinum feature? If so we should test that it is blocked in lower tier.